inlong-distribution/src/main/assemblies/sort-connectors.xml

```diff
@@ -1,159 +1,167 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--

```

Comment:
Suggested not to change the file format.

Reply:
ok. i have fixed it